Automattic/wp-calypso
# Help: Open /help links in Help Center

Related to https://github.com/Automattic/dotcom-forge/issues/8104
Fixes https://github.com/Automattic/dotcom-forge/issues/8104

## Proposed Changes

* These changes add a hook to open an article based on the URL in the Help Center. It will also update `/help` to open the support documentation links in the Help Center instead of opening in a new tab.

## Why are these changes being made?


* Makes it easier for users to open and see support docs without navigating away and losing focus of the current page/content.

## Testing Instructions



* Checkout branch or use live link
* Navigate to `/help`
* Click on any of the default help article cards and verify it opens within the Help Center
* Search for other articles and verify they open within the Help Center
* Change the locale for your account and test `/help` again.
* Verify the community related links still open in a new tab (Theyre displayed below the support docs when you search with a query on `/help`).
* Verify functionality is not missing from the refactored HelpResult component. 
* Verify "Custom Code" and "Free SSL Certification" `/me/developer` links open in Help Center.
